OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -73,7 +73,10 @@ public void PumpInput(Sdl2PlatformWindow device, IInputHandler inputHandler, int
 				switch (e.type)
 				{
 					case SDL.SDL_EventType.SDL_QUIT:
-						Game.Exit();
+						// On macOS, we'd like to restrict Cmd + Q from suddenly exiting the game.
+						if (!(Platform.CurrentPlatform == PlatformType.OSX && mods.HasModifier(Modifiers.Meta)))
```

Comment:
This can be simplified by de-morgan: `if (Platform.CurrentPlatform != PlatformType.OSX || !mods.HasModifier(Modifiers.Meta))`

Reply:
I agree that’s a valid refactor here but I disagree it makes the logic simpler. The code should be as easy as possible for maintainers to understand. “Not macOS with the meta key down” feels easier to understand than the proposed “Not macOS or not the meta key down”. If the extra brackets are your main concern, the logic inside them could be extracted to a well named variable before the `if` statement.

Reply:
Personally I find the latter easier to parse since I don't have to negate at the end, but don't really have a strong opinion here. (The compiler will likely optimize this away regardless.)

Reply:
I also find the latter easier to parse, FWIW. But I also don't think this is particularly important either way.

Reply:
> The code should be as easy as possible for maintainers to understand.
> Personally I find the latter easier to parse
> I also find the latter easier to parse
Can you please apply the fixup then?